src/main/resources/swagger/api-docs.yaml

```diff
@@ -599,6 +599,41 @@ paths:
           description: Submission or Workflow not found
         500:
           description: Internal Error
+          
+  /workspaces/{workspaceNamespace}/{workspaceName}/updateAttributes:
+    get:
+      tags:
+      - Workspaces
+      operationId: updateAttributes
+      summary: |
+        Modify attributes on a workspace.
+      produces:
+      - application/json
+      parameters:
+        - name: workspaceNamespace
+          description: Workspace Namespace
+          required: true
+          type: string
+          in: path
+        - name: workspaceName
+          description: Workspace Name
+          required: true
+          type: string
+          in: path
+        - name: workspaceUpdateJson
+          description: Attribute operations
+          required: true
+          type: Array[AttributeUpdateOperation]
```

Comment:
I'm pretty sure this isn't the correct syntax for the type. I think it is something like this, where `AttributeUpdateOperation` is defined in the definitions:
```
  type: array
    items:
      $ref: "#/definitions/AttributeUpdateOperation"
```